CodeReview/src/items/VReviewComment.cpp

```diff
@@ -36,25 +36,60 @@ namespace CodeReview
 
 DEFINE_ITEM_COMMON(VReviewComment, "item")
 
-VReviewComment::VReviewComment(Visualization::Item* parent, NodeType* node, const StyleType* style)
-	: Super{parent, node, style}
+VReviewComment::VReviewComment(Visualization::Item* parent, NodeType* nodeType, const StyleType* style)
```

Comment:
To avoid the hiding of function `node()`,  but I can just use `VReviewComment::`.